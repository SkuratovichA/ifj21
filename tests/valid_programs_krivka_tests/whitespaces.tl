require "ifj21"
function whitespaces
()
local 
s 
: 
string 
= 
"\x3A0"write(s, "\n")
s = "a\255b"
write(s
, 
"\n")local x:integer=0
-
1write(x)
end whitespaces()
