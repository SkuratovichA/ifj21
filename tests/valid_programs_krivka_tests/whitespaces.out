:0
aÿb
-1